Lesson4/Controllers/ViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ViewController.swift
+//  Lesson4
+//
+//  Created by Алсу Хайруллина on 18.10.2021.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    // Outlets
+    @IBOutlet weak var emailField: UITextField!
+    @IBOutlet weak var passwordField: UITextField!
+    
+    // ViewController life cycle
+    override func viewDidLoad() {
+        if UserDefaults.standard.bool(forKey: "isUserLoggedIn") == true {
```

Comment:
```suggestion
        if UserDefaults.standard.bool(forKey: "isUserLoggedIn") {
```
зачем условие проверять условием?